python/tests/aggregate_tests/test_array_every.py

```diff
@@ -0,0 +1,67 @@
+from .aggtst_base import TstView
+
+
+class aggtst_array_every(TstView):
+    def __init__(self):
+        # Validated on Postgres
```

Comment:
can you also add some documentation to the array.md file which says that comparisons like > work for arrays, and describes how they are computed?